src/constants/ActionTypes.js

```diff
@@ -397,6 +397,7 @@ export default utils.deepKeyMirror({
 			},
 			REMOVE: null,
 			REMOVE_MAP: null,
+			SET_ACTIVE3D: null,
```

Comment:
what about SET_ACTIVE_3D?

Reply:
could be

Reply:
Renamed 5b498f91a2759bc5f6951a3e1b7ece22f7ddca5e